src/execution/sema/sema_builtin.cpp

```diff
@@ -2870,11 +2999,18 @@ void Sema::CheckBuiltinCall(ast::CallExpr *call) {
       CheckBuiltinParamCall(call, builtin);
       break;
     }
+    case ast::Builtin::Chr:
+    case ast::Builtin::CharLength:
+    case ast::Builtin::ASCII:
     case ast::Builtin::Lower:
     case ast::Builtin::Version: {
       CheckBuiltinStringCall(call, builtin);
       break;
     }
+    case ast::Builtin::Position: {
+      CheckBuiltinStringCall(call, builtin);
+      break;
+    }
```

Comment:
Why not put `ast::Builtin::Position` together with the other string functions above?

Reply:
Fixed